G/gmsh/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "gmsh"
+version = v"4.7.1"
+
+# Collection of sources required to build Gmsh
+sources = [
+    GitSource("https://gitlab.onelab.info/gmsh/gmsh.git", "8417af5701df5fb2d4e208424f1477be21f65c3c"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/gmsh
+install_license LICENSE.txt
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DENABLE_BUILD_DYNAMIC=1 ..
```

Comment:
I would recommend to split this with one option per line, using `\` for
Line continuations. Result is easier to read and future tweaks produce easier to read diffs

Reply:
what am I missing in the current commit? Tried it but the last `\` doesn't work for some reason

Reply:
nvm I've found the problematic whitespace